src/content/tools/pub/cmd/pub-token.md

```diff
@@ -26,13 +25,16 @@ https://some-package-repo.com/my-org/my-repo package repository requested authen
 Go to https://some-package-repo.com and log in to obtain your token. 
 ```
 
-The last line is a message the server can provide to help you obtaining a token.
-Some servers might not provide such a message.
+In this last message, the final line gives you the server that can help you
+obtain a token. Not all servers provide such this message.
+
+## Add a new credential
+
+To create a new credential, invoke `dart pub token add`.
 
-## Adding credentials `dart pub token add`
+### Add a credential for the current session
 
-To enter the credentials use `dart pub token add`, 
-and type the credential on stdin.
+At the prompt, type the credential on the command line (`stdin`).
```

Comment:
```suggestion
At the prompt, type the credential on the command line.
```

Since this is covered in the paragraph following the example prompt, I don't think it needs to be mentioned here.